src/main/java/org/openqa/selenium/htmlunit/HtmlUnitWebElement.java

```diff
@@ -131,6 +133,19 @@ public void submit() {
     parent.submit(this);
   }
 
+  @Override
+  public String getDomAttribute(String name) {
+    //TODO
+    return getAttribute(name);
```

Comment:
@rbri I'm not really sure about these 2 methods. Do you have any idea how to implement them?

Reply:
The relevant bit of the spec is here: https://w3c.github.io/webdriver/#get-element-attribute and there are links to the right bits of other specs there, but the TL;DR is that “get element property” is essentially “element.property” in JS.

Reply:
Will try to write a bunch of test cases for this later today.